controllers/admin/AdminOrdersController.php

```diff
@@ -250,6 +251,7 @@ public function renderForm()
         }
 
         $this->context->smarty->assign(array(
+            'customersSearchUrl' => SymfonyContainer::getInstance()->get('router')->generate('admin_customers_search'),
```

Comment:
well, at some point it will be migrated as well, so it will be a little easier to find correct route. On the other hand, i dont know if i can use legacy  `getAdminLink` since page is not enabled.